src/screens/create-skill/form.tsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
```

Comment:
Move this to create-skill/components/create-skill-form.tsx and follow other components file structure